### DISCUSSION TITLE
"Project Best Practices for K8s" meeting series

### DISCUSSION THREAD

**Discussion by akyriako** [cncf/tag-env-sustainability#428] [General]
Hi folks,

Quick question: Are the "Project Best Practices for K8s" meeting series still taking place? because today (03.06.24) the zoom call was empty.

Best regards,
Kyriakos

**Comment by leonardpahlke** [comment]
PTAL @JacobValdemar @xamebax

**Comment by xamebax** [comment]
Hello @akyriako! First of all, apologies for this happening. The project is ongoing, and we would love to have you on our next meeting.

I double-checked that our CNCF ENV TAG weekly meeting calendar had no meeting scheduled for that Monday: https://www.linkedin.com/company/cncf-tag-environmental-sustainability/?miniCompanyUrn=urn%3Ali%3Afs_miniCompany%3A98554943&lipi=urn%3Ali%3Apage%3Ad_flagship3_feed%3B5%2Fjerod0T9mjXyJvINzyWQ%3D%3D

But you're right - yesterday was the first Monday of the month, so the meeting should have been scheduled (it's scheduled for May 6th and July 1st). I'm sorry for this, I'll review our calendar to make sure that all meetings that should be scheduled are scheduled.
